src/extension.ts

```diff
@@ -214,6 +215,13 @@ export function activate(context: vscode.ExtensionContext): void {
 
         node.treeItem.openCdInPortal(node);
     });
+    actionHandler.registerCommand('invalidAppService.Invalid', async (node?: IAzureNode<InvalidWebAppTreeItem>) => {
```

Comment:
What's the purpose of this command? Is it just to show the message? I think this would be confusing for users - a context menu command isn't the best place to show messages. Users expect context menu commands to actually do something

I don't think the message is necessary, but I can help you show the message as a child of the invalidAppService if you want

Reply:
Yeah, I get what you mean. Went ahead and removed it... I don't think any further indicator is necessary :)